### ISSUE TITLE
Lost progress in arcana

### ISSUE THREAD

**Issue by SirEcho123** [ValveSoftware/Dota2-Gameplay#19859]
### Description

just a few minutes ago I got a notification that I received a new arcana item which is pudge arcana. 10/07/24 around 1910.
But I already had that item a long time ago now and I even have some progress on it. I was about 7 more games - estimated- to unlocking the new set.
but now that I got the new item I lost all my progress and I think I only have 1x of the item itself. im so confused. 

in conclusion I received a new item which I already had and progress on it. 
and now I lost all my progress.

### Screenshots

![Screenshot 2024-07-10 192335](https://github.com/ValveSoftware/Dota2-Gameplay/assets/116763500/c8410f6e-3330-4e67-b3ec-068d6539258b)
